metacat-main/src/main/java/com/netflix/metacat/main/api/v1/TagController.java

```diff
@@ -309,11 +320,20 @@ public void removeTableTags(
                     throw new TableNotFoundException(name);
                 }
                 final TableDto oldTable = this.tableService
-                    .get(name, true)
+                    .get(name, GetTableParameters.builder()
+                        .includeInfo(true)
+                        .includeDataMetadata(true)
+                        .includeDefinitionMetadata(true)
+                        .disableOnReadMetadataIntercetor(true)
```

Comment:
The code to use this flag can get complicated. Why didn't we add the interceptor logic at the service layer in metacat-main?

Reply:
This is more of clean-up of the existing API. We should make the get() API call change regardless, since adding flags doesn't scale and the flags can be set in a wrapper which is hidden. 
I am not sure if you understand what you mean by adding the interceptor logic at service layer. We want to put all the interceptor thing in the userMetadataService.